docs/devices_api.yml

```diff
@@ -82,7 +82,7 @@ definitions:
         type: string
         description: Attribute description.
       value:
-        type: object
+        type: string
```

Comment:
Isn't it a bit misleading? it can be many different types.
I know of few more places we will need to use 'object' without specific structure as we are going to display metadata of artifacts which are allowed to have any structure defined by the user.